.circleci/config.yml

```diff
@@ -188,10 +188,8 @@ workflows:
           requires:
             - lint
       - pypy3:
-          filters:
-            branches:
-              only:
-                - 0.20.X
+          requires:
+            - lint
```

Comment:
Do we really want to run the pypy3 job on all pull requests? Or keep it disabled for the PR and only run for the cron scheduled nightly builds?

Reply:
Furthermore, running the `pypy3` job as part of a workflow named`build-doc-and-deploy` is really counter intuitive.

Reply:
I think we should remove the pypy3 job from the `build-doc-and-deploy` workflow.

Reply:
Fine with me. At least it was to check that everything worked in this PR.

Reply:
I'm happy not to have it in `build-doc-and-deploy`, but this bites us almost every time at a release, should we not have at least one PyPy test on PRs?